content/ai/concepts/chatgpt/chatgpt.md

```diff
@@ -0,0 +1,103 @@
+---
+Title: 'ChatGPT'
+Description: 'ChatGPT is a publically available web-based chatbot and API interface created by OpenAI. It connects to a large language model (LLM) called GPT (Generative Pre-Trained Transformer). GPT includes some of the largest models ever created. After the model training, there is further fine-tuning to improve its generated responses.'
+Subjects:
+  - 'Machine Learning'
+  - 'Data Science'
+  - 'Large Language Model'
+  - 'Transformers'
+Tags:
+  - 'AI'
+  - 'Transformers'
+  - 'Embeddings'
+  - 'Prompts'
+  - 'Prompt Engineering'
+  - 'Attention'
+CatalogContent:
+  - 'machine-learning'
+  - 'paths/data-science'
+---
+
+`ChatGPT` is a publically available web-based chatbot and API interface created by `OpenAI`. It connects to a large language model (LLM) called GPT (Generative Pre-Trained Transformer). GPT includes some of the largest models ever created. After the model training, there is further fine-tuning to improve its generated responses.
+
+## How ChatGPT Works
+
+`ChatGPT` is part of a new group of LLM models which are now made available by a technology called `Transformers`. Other notable and similar LLM models include `BERT` used by Google to understand user searches, `DALL-E` which can generate images, and `GPT-Neo` an promising open source LLM.
+
+ChatGPT is the packaging of a set of trained LLMs into a chatbot and developer friendly interface. This has created a system where there is low barrier to entry and almost everyone can start using the LLM.
+
+### What is the relation to Internet Search?
+
+`ChatGPT` is trained by using vast quantities of data from the Internet. However LLMs are different from Internet search engines:
+
+- No ongoing daily web crawlers and updates
+- Data can be out of date. `ChatGPT 3` uses data from 2021.
+- The size of the data trained in the model
+- ChatGPT uses a type of fine-tuning called Reinforcement Learning from Human Feedback (RLHF)
+- ChatGPT-specific approaches including `prompts`, `embeddings`, and parameters such as `temperature`.
+
+## History of ChatGPT
+
+There were several generations of AI technologies that have led up to ChatGPT.
+
+### Feed Forward Networks
+
+Artificial Neural Networks (ANNs) are a class of computational functions inspired by how neurons work in the human brain. Neurons are arranged in layers and training occurs by guiding the network to know correct from incorrect answers. These are called feed-forward networks because signals move forward through the system.
```

Comment:
```suggestion
**Artificial Neural Networks** (ANNs) are a class of computational functions inspired by how neurons work in the human brain. Neurons are arranged in layers and training occurs by guiding the network to differentiate between correct and incorrect answers. These are called feed-forward networks because signals move forward through the system.
```

Reply:
Resolved issues